modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/nhttp/HttpCoreNIOSender.java

```diff
@@ -153,12 +153,9 @@ public void init(ConfigurationContext cfgCtx, TransportOutDescription transportO
 
         connpool = new ConnectionPool();
 
-        proxyConfig = new ProxyConfigBuilder().parse(transportOut).build();
-        if (log.isInfoEnabled() && proxyConfig.getProxy() != null) {
-            log.info("HTTP Sender using Proxy " + proxyConfig.getProxy() + " bypassing " + 
-                proxyConfig.getProxyBypass());
-        }
-        
+        proxyConfig = new ProxyConfigBuilder().build(transportOut);
+        log.info(proxyConfig.logProxyConfig());
+
         Parameter param = transportOut.getParameter("warnOnHTTP500");
```

Comment:
Completely agree. One "significant" point I missed adding in my earlier comment is that, 
"....... improve these additional bits as well..." as part of the JIRA but not necessarily within the same commit. I'd looked for even for any JIRA or something created around adding proper constants upon those additional fields, but found none. Hence, the comment.